prime-router/docs/design/proposals/translation-template-test-plan.md

```diff
@@ -0,0 +1,57 @@
+# Translation Template Test Plan
+
+## Problem Statement
+As a Reportstream engineer, I want tests that verify that the changes I make to translations will not break essential 
```

Comment:
What spec? I think we should clarify that our test plan should help us verify that our mappings (templates) implement the HL7v2 mapping inventory with NIST 2.5.1? @mkalish. If we are going to say "the spec" throughout the doc, we need to clarify what exactly that is.